sbt-app/src/sbt-test/project/semanticdbEnabled/test

```diff
@@ -0,0 +1,25 @@
+> compile
+$ exists target/**/scala-2.12*/**/meta/META-INF/semanticdb/matrix/src/main/scala/foo/Compile.scala.semanticdb
+$ exists target/**/scala-3*/**/meta/META-INF/semanticdb/matrix/src/main/scala/foo/Compile.scala.semanticdb
+
+> clean
+-$ exists target/**/scala-2.12*/**/meta/META-INF/semanticdb/matrix/src/main/scala/foo/Compile.scala.semanticdb
+-$ exists target/**/scala-3*/**/meta/META-INF/semanticdb/matrix/src/main/scala/foo/Compile.scala.semanticdb
+> compile
+$ exists target/**/scala-2.12*/**/meta/META-INF/semanticdb/matrix/src/main/scala/foo/Compile.scala.semanticdb
+$ exists target/**/scala-3*/**/meta/META-INF/semanticdb/matrix/src/main/scala/foo/Compile.scala.semanticdb
+
+> set LocalProject("matrix2_12") / Compile / semanticdbTargetRoot := (LocalProject("matrix2_12") / Compile / target).value / "custom212"
+> set LocalProject("matrix3") / Compile / semanticdbTargetRoot := (LocalProject("matrix3") / Compile / target).value / "custom3"
```

Comment:
Unfortunately, I ended up using that syntax because
- bare settings don't work with `set`
- matrix projects can't be addressed by their name (but only by their ID??)